src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -478,14 +474,30 @@ def fuzz(self,
     """LibFuzzerCommon.fuzz override."""
     self._test_ssh()
 
-    #TODO(flowerhack): Pass libfuzzer args (additional_args) here
+    logs.log('SSH works; sync corpus directories')
+    # Push corpus directories to the device.
+    data_dst = self.fuzzer.data_path('corpus')
+    for corpus in corpus_directories:
+      for corpfile in os.listdir(corpus):
```

Comment:
s/corpfile/corpus_file/

Reply:
changed to corpus_dir since that seems more accurate (it's a list of directories, not files, right?)